.github/workflows/CI.yml

```diff
@@ -79,6 +85,187 @@ jobs:
           which pip
           python -m pip install -U pip
           python -m pip install -U --prefix "${INSTALLDIR}/conda" -r requirements-dev.txt
+          cd "${HOME}"
+          zip -vr taucmdr.zip ${INSTALLDIR##*/} -x "taucmdr/system/src/*" -x "taucmdr/conda/*"
+
+      - name: Upload TAU Commander install log
+        if: always()
+        uses: actions/upload-artifact@v1
+        with:
+          name: taucmdr-tau-minimal-debug-log-${{ matrix.os }}
+          path: ${{ steps.info.outputs.home_dir }}/.local/taucmdr/debug_log
+
+      - name: Upload TAU Commander minimal install
+        if: always()
+        uses: actions/upload-artifact@v1
+        with:
+          name: taucmdr-installation-${{ matrix.os }}
+          path: ${{ steps.info.outputs.home_dir }}/taucmdr.zip
+
+  Install-taucmdr-full:
+    needs: Install-taucmdr-minimal
```

Comment:
A taucmdr minimal TAU installation (normal case) happens and the python `conda` dir is manipulated (saved and restored) via caching, as is the `system/src` directory. The rest of the installation is zipped into an archived and uploaded. Then, a unit test and TAU=full job depend on this first job, download and restore the taucmdr installation, and proceed in parallel: unit testing happens (~40 minutes) while a `TAU=full` install happens (> 1 hour). Both upload debug logs for later inspection and results of `tau_validate`